packages/react/src/elements/components/GlobalNav/SideNav/Module.js

```diff
@@ -8,8 +8,7 @@ class Module extends Component {
     onClick: PropTypes.func.isRequired
   };
 
-  handleClick = event => {
-    event.preventDefault();
```

Comment:
This event intercept was in place to prevent the page from scrolling to the top when clicking a module or submodule. However, this also prevents navigating to a valid link.

To address both scenarios we can let the event propagate and remove the default `"#"` href.

Reply:
👏